presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -472,6 +473,27 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
     }
 
     @Override
+    public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column)
+    {
+        RaptorTableHandle table = (RaptorTableHandle) tableHandle;
+        RaptorColumnHandle raptorColumn = (RaptorColumnHandle) column;
+
+        List<TableColumn> existingColumns = dao.listTableColumns(table.getSchemaName(), table.getTableName());
+        if (existingColumns.size() == 1) {
+            throw new PrestoException(NOT_SUPPORTED, "Dropping last column is not supported");
+        }
+        long maxColumnId = existingColumns.stream().map(s -> s.getColumnId()).max(Comparator.comparing(i -> i)).get();
```

Comment:
Something of the form `s -> s.getId()` can be a method reference. IntelliJ has an inspection/intention for this.

Finding the max is better using `LongStream`:
```java
stream().mapToLong(TableColumn::getColumnId).max().getAsLong()